### DISCUSSION TITLE
localePath link for [...slug.vue]?

### DISCUSSION THREAD

**Discussion by notflip** [nuxt-modules/i18n#2704] [Q&A]
When working with a dynamic catch-all page (which is populated with data from my Directus CMS). How can we make sure we have a correct `localePath` link to that page from within my navigation?

With a static page (contact.vue and about.vue), this works, but **what is the router name when using [...slug].vue?**
```
    <nuxt-link :to="localePath({name: 'about'})">{{ $t('router.about') }}</nuxt-link>
    <nuxt-link :to="localePath({name: 'contact'})">{{ $t('router.contact') }}</nuxt-link>
```

Thanks!

**Comment by rchl** [comment]
You can check generated route names in `.nuxt/router.js`

**Reply by notflip** [reply]
@rchl Thanks for getting back this quick, are you sure that's the right path? I don't see a `.nuxt/router.js` file, not when running `yarn build` or when running `yarn dev`. Thanks! 

<img width="257" alt="Screenshot 2024-01-17 at 17 25 14" src="https://github.com/nuxt-modules/i18n/assets/4146384/c07bee41-20fc-4e44-a9bf-da8d7b963bea">

**Reply by notflip** [reply]
I managed to output the list of routes and their names inside a vue file using `$router.options.routes"`

**Reply by rchl** [reply]
Right. I was thinking of Nuxt 2 here. Not sure about Nuxt 3 equivalent of that.

**Reply by BobbieGoede** [reply]
Another way to see the generated names is to check the routes tab in Vue Devtools or pages tab in Nuxt Devtools.
